install-requirements.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+printf "=================== INSTALL CONCENT_API REQUIREMENTS ====================\n"
+pip install -r concent_api/requirements.lock
+
```

Comment:
don't we need `requirements-development.txt` as well. Even if they are not updated so often, I'm afraid, in e.g. two months we will remember only that we have one script with **all** requirements and we will not update `requirements-development.txt`.. I think it should be added, even if `requirements-development.txt` will be unchanged nothing bad will happen.

Reply:
Added. Done

Reply:
Added. Done